src/ripple/nodestore/backend/CassandraFactory.cpp

```diff
@@ -670,7 +670,7 @@ class CassandraBackend : public Backend
         // confirmed persisted. Otherwise, it can become deleted
         // prematurely if other copies are removed from caches.
         std::shared_ptr<NodeObject> no;
-        NodeStore::EncodedBlob e;
+        std::optional<NodeStore::EncodedBlob> e;
```

Comment:
It's because I don't want `EncodedBlob` to have a default constructor, or to be forced to have a special "empty" `EncodedBlob` state.

Seen in context, the performance overhead of `std::optional` is minimal and it helps keep the API clean and simple.